tests/kernel/boot_ltp.pm

```diff
@@ -37,6 +37,11 @@ sub run {
         assert_script_run("lsblk -la; export LTP_BIG_DEV=$block_dev");
     }
 
+    # check kGraft patch if KGRAFT=1
+    if (check_var('KGRAFT', '1')) {
```

Comment:
As '1' can be treated as true, maybe we can use get_var and then we can transform it to one liner. But it's matter of taste.